src/builtin-addons/core/script-definition-provider.ts

```diff
@@ -185,9 +194,19 @@ export default class CoreScriptDefinitionProvider {
     } else if (isImportSpecifier(astPath)) {
       logInfo(`Handle script import for Project "${project.name}"`);
       const pathName: string = ((astPath.parentFromLevel(2) as unknown) as t.ImportDeclaration).source.value;
+      const pathPaths = pathName.split('/');
+      const maybeAppName = pathPaths.shift();
 
       project.roots.forEach((projectRoot) => {
-        const potentialPaths = this.guessPathForImport(projectRoot, uri, pathName) || [];
+        const pkg = getPackageJSON(projectRoot);
```

Comment:
seems too heavy, could we use Project.name + filter from Project.addonsMeta.[].name here?